TheMET/TheMET/ViewModels/AboutMETViewModel.swift

```diff
@@ -43,8 +44,73 @@ class AboutMETViewModel {
         let aboutAppNavigationController = UINavigationController(rootViewController: aboutAppViewController)
         presentingController.present(aboutAppNavigationController, animated: true)
     }
+    
+    func topLocatonViewDidTap() {
+        self.locationViewDidTap(latitude: self.topMapViewData.latitude, longitude: self.topMapViewData.longitude, name: NSLocalizedString("about_met_the_met_fifth_avenue_title_lable", comment: ""), address: "The+Metropolitan+Museum+of+Art,+1000+Fifth+Avenue+New+York,+NY+10028")
```

Comment:
Here and in another method:
- I'd recommend to repack your current static data:
-- Introduce extension to `CLLocationCoordinate2D` with two static coordinates: `fifthAvenue` and `cloisters`
-- Use the static coordinates in the extensions for `AboutMETViewModel.MapViewData`
-- pass `coordinate: CLLocationCoordinate2D` in all methods instead of separate `latitude` and `longitude`